FinalTask/FinalTaskGM/FinalTaskGM/Tests/BaseTest.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Allure.Commons;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Remote;
+using System;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.IO;
+
+namespace FinalTaskGM.Tests
+{
+    public class BaseTest : AllureReport
+    {
+        public IWebDriver driver { get; set; }
+        protected string pathToScreenshots;
+        public string TestService { get; }
+
+        public BaseTest(string type)

```

Comment:
You don't need parameterized constructor. Just initialize TestService variable in place of declaration.